stubs/netaddr/netaddr/ip/__init__.pyi

```diff
@@ -148,14 +148,14 @@ class IPRange(BaseIP, IPListMixin):
     def cidrs(self) -> list[IPNetwork]: ...
 
 def iter_unique_ips(*args: IPRange | _IPNetworkAddr) -> Iterator[IPAddress]: ...
-def cidr_abbrev_to_verbose(abbrev_cidr: str | SupportsInt | SupportsIndex) -> str: ...
+def cidr_abbrev_to_verbose(abbrev_cidr: ConvertibleToInt) -> str: ...
```

Comment:
This change seems reasonable to me; the function at runtime is deliberately very permissive in what it accepts